### DISCUSSION TITLE
Step 2 won't update to the next step after refreshing

### DISCUSSION THREAD

**Discussion by kakrakak** [skills/.github#281] [Code with Copilot]
I am following instructions from the 30 day challenge

**Comment by sinsukehlab** [comment]
Do you mean the "Introduction to GitHub" course? You should have edited and committed to the `my-first-branch` branch, not the `main` branch.

**Reply by T095T** [reply]
So i need to create a new branch named 'my_first_branch' linked to main and edit in the codespace for it?
Am i correct?

**Reply by sinsukehlab** [reply]
You should first create a new branch named "`my-first-branch`" (case-sensitive, hyphenated) from the `main` branch. You don't have to use Codespaces in the "Introduction to GitHub" course. You can just edit in situ.

**Reply by sinsukehlab** [reply]
@T095T You're mistaking the "**Code with GitHub Copilot**" course for "**Introduction to GitHub**" course. You just commit directly to the `main` branch in the "**Code with GitHub Copilot**" course.

**Comment by stockers** [comment]
I have same issue, Did Step 2, refreshed browser, stuck on step 2. How to advance?  It's all a bit of a faff..

**Reply by sinsukehlab** [reply]
@stockers @T095T Please open a different Q&A conversation. You have an issue on the "**Code with GitHub Copilot**" course, not on the "**Introduction to GitHub**" course.

**Reply by stockers** [reply]
My reply is to Kakrakak q in "code with colpilot" discussion which is the title at top of this thread. Turns our (by diving into the build action code) that a git hub action is looking for the exact case sensitive comment in the pushed js file in order to then update the read me file to step 3.

**Reply by sinsukehlab** [reply]
@stockers
@kakrakak hasn't generated a "**Code with Copilot**" repo (a public one, at least). The date when the question was asked suggests that the course where they got stuck should be "**Introduction to GitHub**".
I'm afraid that you have got off topic. The category was Q&A when the question was asked.

**Comment by sinsukehlab** [comment]
@cmwilson21 This discussion was on the "Introduction to GitHub" course although the two have got off topic. Could you reclassify this discussion as "Introduction to GitHub"?
